### PR TITLE
Add first-in-series support

### PR DESCRIPTION
**Pull request by mt-clearhaus**
This is a part of the token acceptance project: https://github.com/clearhaus/issues-pci/issues/3011

### CODE REVIEW

**Comment by mt-clearhaus** [comment]
I'm not particularly fond of my solution with preprocessing in the start script, however, I don't want to spend time on HTML and JS. So don't ask me to do that :slightly_smiling_face:

**Comment by mt-clearhaus** [comment]
See additional tests in https://github.com/clearhaus/issues-pci/issues/3163.

**Comment by mt-clearhaus** [comment]
Setting this ready for review as I would appreciate some input, or perhaps some assistance if someone wants to replace the shell script preprocessing with some nifty JS.

At this point I don't imagine spending much more time on this.

**Comment by mt-clearhaus** [comment]
> At this point I don't imagine spending much more time on this.

Unless Ben West comes back as some point saying that we sent incorrect recurring payment details to trigger a COF/merchant/network token.

**Comment by brorbw** [comment]
We should probably have the service serve both at the same time rather than having it do only one thing at a time. My reasoning is that we can only bind one service to `applepaymerchant.clrhs.dk` and thus having to take the service down to switch between them might be cumbersome if you end in a scenario where you would want to test both. 
I would much rather have something like `applepaymerchant.clrhs.dk/authroization` and `applepaymerchant.clrhs.dk/first`. However I understand if you don't want to spend more time on this and I don't feel strongly about it

**Comment by mt-clearhaus** [comment]
@brorbw Fully agree :slightly_smiling_face:  I think I'll ditch the preprocessing and just do a "crude" serving of almost similar files in two different directories. Copy-pasted lines/code is ugly, however, the preprocessing with a prompt is worse, IMO.

**Comment by mt-clearhaus** [comment]
:tada: 

IMG_20221103_150018975_2

**Comment by mt-clearhaus** [comment]
Pro tip: If something looks iffy then go to Settings --> Safari --> Clear cache. Your device might have cached old JS and/or CSS. I have made sure nothing will be cached going forward.

**Comment by mt-clearhaus** [comment]
First-in-series part tested successfully by @brorbw in https://clearhaus.slack.com/archives/C049G9KDEN4/p1667481845795879.

**Comment by mt-clearhaus** [comment]
So I ended up doing a bit of JS anyway :see_no_evil: :sweat_smile:

**Comment by mt-clearhaus** [comment]
From @brorbw's test:

IMG_1978

File: build/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,21 @@
 FROM debian:bullseye-slim
 
-LABEL Author="kse@clearhaus.com" \
-      Maintainer="Clearhaus"
+LABEL Author="operations.pci@clearhaus.com" \
+      Maintainer="Clearhaus PCI"
```
Comment (mt-clearhaus) [inline build/Dockerfile:5]:
```suggestion
```

File: public/index.js [hunk 2]
```diff
@@ -21,38 +22,64 @@ document.addEventListener('DOMContentLoaded', () => {
...
+        amount: '1.00',
+        paymentTiming: 'recurring',
+        recurringPaymentStartDate: new Date(),
+        recurringPaymentIntervalUnit: 'month',
+        recurringPaymentIntervalCount: 3,
+      },
+      managementURL: 'https://applepaymerchant.clrhs.dk'
+    };
+  } else {
+    debug('Invalid payment type; returning');
```
Comment (mt-clearhaus) [inline public/index.js:79]:
```suggestion
    debug('Invalid payment type');
```

**Comment by mt-clearhaus** [comment]
Adding @kbadk explicitly as reviewer as he is on the task in the planning markdown file :slightly_smiling_face:

**Review by kbadk — APPROVED** [review]
Looks good! 👍🏻

**Pull request merged**